src/test/java/com/softserveinc/ita/CheckoutPageTest.java

```diff
@@ -11,25 +11,28 @@
 
 public class CheckoutPageTest extends TestRunner {
 
-    protected HomePage homePage = new HomePage();
-    protected ProductPage productPage = new ProductPage();
-    protected CheckoutPage checkoutPage = new CheckoutPage();
+    private final HomePage homePage = new HomePage();
 
     @Test
     public void verifyCheckoutPageFunctionalityTest() {
-        homePage.closeAdBanner();
-        var lastProductOfSection = homePage.getLastProductOfSection("Акційні пропозиції");
+        homePage
+                .closeAdBanner()
+                .openLastProductOfSection("Акційні пропозиції");
+
+        var lastProductOfSection = homePage.getLastProductTitleOfSection("Акційні пропозиції");
+        var productPage = new ProductPage();
 
         assertThat(lastProductOfSection)
                 .as("Last product of section should be opened")
-                .isEqualTo(productPage.getProductName());
+                .isEqualTo(productPage.getProductTitle());
 
         productPage
                 .addProductToBasket()
```

Comment:
have you checked your tests after merging master? I see that this method renamed

Reply:
yeah i did